docs/sphinx/user_guide/tutorial/add_vectors.rst

```diff
@@ -70,25 +72,28 @@ Alternatively, RAJA provides a *loop execution* policy::
   RAJA::loop_exec
 
 This policy allows the compiler to generate optimizations, such as SIMD if 
-compiler heuristics suggest that it is safe to do so, but the optimizations
-are not forced.
+compiler heuristics suggest that it is safe to do so and potentially 
+beneficial for performance, but the optimizations are not forced.
 
 To run the kernel with OpenMP multi-threaded parallelism on a CPU, we use the
 ``RAJA::omp_parallel_for_exec`` execution policy:
 
 .. literalinclude:: ../../../../examples/tut_add-vectors.cpp
                     :lines: 137-139
 
+This will distribute the loop iterations across CPU threads and run the 
+loop over threads in parallel.
+
 Finally, to run the kernel on a CUDA GPU device, we use the ``RAJA::cuda_exec``
 policy:
 
 .. literalinclude:: ../../../../examples/tut_add-vectors.cpp
                     :lines: 151-154
 
 Note that the CUDA execution policy type accepts a template argument 
-``CUDA_BLOCK_SIZE``, which we've defined to be an integral value of 256
-in the example. This means that each CUDA thread block that is launched to 
-execute the kernel will have 256 threads in the block. The thread block size
+``CUDA_BLOCK_SIZE``, which you can change in the example code. This value
+specifies that each CUDA thread block that is launched to execute the kernel 
+will have the given number threads in the block. The thread block size
```

Comment:
Maybe should be re-worded to something like: "This value specifies the number of threads per CUDA thread block which are launched to execute the kernel."

Reply:
Fixed